scripts/move_to_spark_2.sh

```diff
@@ -2,10 +2,12 @@
 
 set +x
 
-svp="\${scala\.version\.prefix}"
+svp="\${scala.version.prefix}"
 substitution_cmd="s/_$svp/-spark2_$svp/g"
 
 find . -name "pom.xml" -exec sed \
+    -e "/utils-/ s/_2\.10/-spark2_2.10/g" \
+    -e "/adam-/ s/_2\.10/-spark2_2.10/g" \
```

Comment:
That line of thought was what led me to delete it originally. However, yeah, we're intentionally not using SVP in the artifact names. That is what #1254 broke. This looks to be passing but I'd like to go over the CI logs before we merge.